src/Installers/Rpm/Runtime.Rhel.rpmproj

```diff
@@ -0,0 +1,13 @@
+<!--
+  This project produces a RPM RHEL.7 installer for Linux x64 platforms only.
+-->
+<Project DefaultTargets="Build">
+  <Import Project="Rpm.props" />
+
+  <PropertyGroup>
+    <RpmPackageInstallRoot>/opt/rh/rh-dotnet20/root/usr/lib64/dotnet/</RpmPackageInstallRoot>
```

Comment:
Hmm we need to check if this path is the correct one to use for 2.2. cc @leecow ?

Reply:
This value is also what we use in 2.1. https://github.com/aspnet/AspNetCore/blob/release/2.1/build/SharedFxInstaller.targets#L122

Reply:
I asked about this in an email thread as well and I don't recall getting a definitive answer either.